Update doc for SE-19066 (#160)
mulesoft/docs-monitoring: modules/ROOT/pages/logs-using.adoc

filter, and analyze your logs. You can use aggregated logs in conjunction with
monitoring to help identify issues and investigate failures quickly.

> == Warnings
> 
> ==== For On-Premises Servers With Customized Logging
> 
> Logs produced by apps with customized logging (by changing the pattern layout in the `log4j2.xml` file) might not get indexed correctly into Anypoint Monitoring. To ensure correct indexing, it is recommended to use Mule default pattern layout, which is one of the following: 
> 
> * `<PatternLayout pattern="%-5p %d [%t] [event: %X{correlationId}] %c: %m%n" />`, or
> * `<PatternLayout pattern="%d [%t] %-5p %c - %m%n" />`
> 
> ==== For Applications Deployed to Cloudhub and Runtime Fabric With Customized Time Zone
> 
> Do not override the default JVM timezone(it should always be UTC). Changing the timezone to a local timezone will cause logs indexed into Anypoint Monitoring with an incorrect timestamp. 

Reviewer: ```suggestion
[CAUTION]
==== 

*For on-premises servers with customized logging:*

Logs produced by apps with customized logging (by changing the pattern layout in the `log4j2.xml` file) might not get indexed correctly into Anypoint Monitoring. To ensure correct indexing, use the Mule default pattern layout, which is one of the following: 

* `<PatternLayout pattern="%-5p %d [%t] [event: %X{correlationId}] %c: %m%n" />`, or
* `<PatternLayout pattern="%d [%t] %-5p %c - %m%n" />`

*For applications deployed to Cloudhub and Anypoint Runtime Fabric with customized time zones:*

Do not override the default JVM timezone for applications deployed to Cloudhub and Anypoint Runtime Fabric with customized time zones. The timezone must be UTC. Changing the timezone to a local timezone causes logs to be indexed in Anypoint Monitoring with incorrect timestamps.

====
```

* `<PatternLayout pattern="%-5p %d [%t] [event: %X{correlationId}] %c: %m%n" />`, or
* `<PatternLayout pattern="%d [%t] %-5p %c - %m%n" />`

==== Note For Applications Deployed to Cloudhub and Runtime Fabric With Customized Time Zone

> Do not override the default JVM timezone(it should always be UTC). Changing the timezone to a local timezone will break Anypoint Monitoring.

Reviewer 2: `Changing the timezone will cause logs indexed into Anypoint Monitoring with incorrect timestamp.`

> Do not override the default JVM timezone(it should always be UTC). Changing the timezone to a local timezone will break Anypoint Monitoring.

Reviewer: ```suggestion
[[WARNING]] 

Do not override the default JVM timezone for applications deployed to Cloudhub and Anypoint Runtime Fabric with customized time zones. The timezone must be UTC. Changing the timezone to a local timezone causes logs to be indexed in Anypoint Monitoring with incorrect timestamps.
```
